src/coreclr/binder/assemblybindercommon.cpp

```diff
@@ -215,6 +215,8 @@ namespace BINDER_SPACE
                     *PeKind = peARM64;
                 else if (dwImageType == IMAGE_FILE_MACHINE_AMD64)
                     *PeKind = peAMD64;
+                else if (dwImageType == IMAGE_FILE_MACHINE_LOONGARCH64)
```

Comment:
@jkotas 
Now the compiling errors are only on windows.
Is this dependent on the windows ?

Reply:
It's only used on windows,  guessed by `src/coreclr/binder/CMakeLists.txt` .